### ISSUE TITLE
Contact Sensor state reported in % in log file

### ISSUE THREAD

**Issue by Krillle** [ilcato/homebridge-fibaro-home-center#164]
This device is a contact sensor (Fibaro Smart Implant). The logifle reports the "open" state as 1%. 

[11/11/2023, 10:12:38] [FibaroHC] Kosmetikspiegel 33.2 [36]: Off 
**[11/11/2023, 10:12:43] [FibaroHC] Starkwind  [23]: 1 %**
[11/11/2023, 10:12:48] [FibaroHC] Temperatur 44.0.1 [46]: 22.2 C

<img width="530" alt="Bildschirmfoto 2023-11-11 um 10 22 26" src="https://github.com/ilcato/homebridge-fibaro-home-center/assets/2863110/b0a80e5c-a904-4697-8d82-dc7eb3692507">

**Comment by mkz212** [comment]
Can you please attach response from API for this device?  Instructions here: #157

**Comment by Krillle** [comment]
```
{
  "id": 23,
  "name": "Starkwind ",
  "roomID": 9,
  "type": "com.fibaro.windowSensor",
  "baseType": "com.fibaro.doorWindowSensor",
  "enabled": true,
  "visible": true,
  "isPlugin": false,
  "parentId": 21,
  "remoteGatewayId": 0,
  "interfaces": [
    "fibaroAlarm",
    "fibaroAlarmArm",
    "fibaroBreach",
    "tamper",
    "zwave",
    "zwaveSceneActivation"
  ],
  "properties": {
    "parameters": [
      {
        "id": 1,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 2,
        "value": 0
      },
      {
        "id": 2,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 2,
        "value": 0
      },
      {
        "id": 3,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 1,
        "value": 0
      },
      {
        "id": 4,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 1,
        "value": 0
      },
      {
        "id": 5,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 1,
        "value": 0
      },
      {
        "id": 6,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 1,
        "value": 0
      },
      {
        "id": 7,
        "lastReportedValue": 255,
        "lastSetValue": 255,
        "size": 1,
        "value": 255
      },
      {
        "id": 8,
        "lastReportedValue": 255,
        "lastSetValue": 255,
        "size": 1,
        "value": 255
      },
      {
        "id": 9,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 1,
        "value": 0
      },
      {
        "id": 10,
        "lastReportedValue": 20,
        "lastSetValue": 20,
        "size": 1,
        "value": 20
      },
      {
        "id": 11,
        "lastReportedValue": 200,
        "lastSetValue": 200,
        "size": 1,
        "value": 200
      },
      {
        "id": 12,
        "lastReportedValue": 8,
        "lastSetValue": 8,
        "size": 1,
        "value": 8
      },
      {
        "id": 13,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 1,
        "value": 0
      },
      {
        "id": 14,
        "lastReportedValue": 0,
        "lastSetValue": 0,
        "size": 1,
        "value": 0
      }
    ],
    "pollingTimeSec": 0,
    "zwaveCompany": "Fibargroup",
    "zwaveInfo": "3,3,52",
    "zwaveVersion": "2.1",
    "alarmDelay": "0",
    "alarmExclude": "true",
    "alarmTimeTimestamp": "0",
    "armConditions": "{\"auto\":false,\"devices\":[{\"id\":23,\"propertyName\":\"value\",\"propertyValue\":\"0\"}],\"time\":0}",
    "armConfig": "0",
    "armDelay": "0",
    "armError": "{}",
    "armTimeTimestamp": "0",
    "armed": "false",
    "categories": "[\"security\"]",
    "configured": true,
    "dead": "false",
    "deadReason": "",
    "deviceControlType": "0",
    "deviceIcon": "1008",
    "emailNotificationID": "0",
    "emailNotificationType": "0",
    "endPointId": "1",
    "fibaroAlarm": "false",
    "lastBreached": "1699694889",
    "log": "",
    "logTemp": "",
    "manufacturer": "",
    "markAsDead": "true",
    "model": "",
    "nodeId": "5",
    "parametersTemplate": "85",
    "productInfo": "1,15,5,1,16,2,2,1",
    "pushNotificationID": "197",
    "pushNotificationType": "0",
    "remoteGatewayId": "0",
    "saveLogs": "true",
    "sceneActivation": "21",
    "serialNumber": "",
    "smsNotificationID": "0",
    "smsNotificationType": "0",
    "tamper": "false",
    "useTemplate": "true",
    "userDescription": "Zentrale Jalousiensteuerung bei starkem Wind",
    "value": "false"
  },
  "actions": {
    "forceArm": 0,
    "meetArmConditions": 0,
    "reconfigure": 0,
    "setArmed": 1
  },
  "created": 1698659712,
  "modified": 1698659712,
  "sortOrder": 117
}
```

<img width="768" alt="Bildschirmfoto 2023-11-11 um 11 10 34" src="https://github.com/ilcato/homebridge-fibaro-home-center/assets/2863110/661eea70-8407-4985-a6ac-92c75b88c166">

**Comment by mkz212** [comment]
@Krillle Thank You. I just add PR to fix this.

**Comment by mkz212** [comment]
It should be fixed in 1.5.0 beta 5.
